draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -740,7 +740,7 @@ A chunk contains either control information or user data.</t>
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 </artwork>
 <t>INIT, INIT ACK and SHUTDOWN COMPLETE chunks MUST NOT be bundled
-into one SCTP packet.
+with any other chunks into an SCTP packet.
```

Comment:
I'm not a native speaker, but wouldn't that allow the bundling of an INIT chunk with an SHUTDOWN COMPLETE chunk? So wouldn't it be better to use "MUST NOT be bundled with any chunk into an SCTP packet"?